pkg/serverless/logs/scheduler.go

```diff
@@ -6,50 +6,38 @@
 package logs
 
 import (
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery"
+	"github.com/DataDog/datadog-agent/pkg/logs"
 	"github.com/DataDog/datadog-agent/pkg/logs/config"
-	"github.com/DataDog/datadog-agent/pkg/logs/schedulers"
+	"github.com/DataDog/datadog-agent/pkg/logs/schedulers/channel"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
-// chanSource is the LogSource installed by this scheduler.
-var chanSource *config.LogSource
-
-// Scheduler is a logs-agent Scheduler that only manages one source, connected to
-// the serverless logs channel.
-type Scheduler struct {
-	logsChan  chan *config.ChannelMessage
-	extraTags []string
-}
-
-var _ schedulers.Scheduler = &Scheduler{}
-
-// NewScheduler creates a new Scheduler.
-func NewScheduler(logsChan chan *config.ChannelMessage, extraTags []string) schedulers.Scheduler {
-	return &Scheduler{logsChan, extraTags}
-}
-
-// Start implements schedulers.Scheduler#Start.
-func (s *Scheduler) Start(sourceMgr schedulers.SourceManager) {
-	log.Debug("Adding AWS Logs collection source")
+// logsScheduler is the current logs-agent scheduler managing the log-source for
+// serverless's messages
```

Comment:
What do you mean by `serverless's messages` here?

Reply:
The ChannelMessage values